diff -Naur vstuff-1.1.2/libkstreamer/Makefile.am vstuff-1.1.2.oden/libkstreamer/Makefile.am
--- vstuff-1.1.2/libkstreamer/Makefile.am	2008-11-05 15:49:11.000000000 +0100
+++ vstuff-1.1.2.oden/libkstreamer/Makefile.am	2008-11-26 16:06:16.000000000 +0100
@@ -61,7 +61,7 @@
 	then mv -f "$(DEPDIR)/libkstreamer_la-pd_grammar.Tpo" "$(DEPDIR)/libkstreamer_la-pd_grammar.Plo"; else rm -f "$(DEPDIR)/libkstreamer_la-pd_grammar.Tpo"; exit 1; fi
 
 libkstreamer_la_LIBADD = libkstreamer_la-pd_grammar.lo
-libkstreamer_la_LDFLAGS = -module -version-info 3:0:0 -no-undefined
+libkstreamer_la_LDFLAGS = -version-info 3:0:0 -no-undefined
 
 AM_CFLAGS = -Wall -D_REENTRANT -D_GNU_SOURCE -D_LIBKSTREAMER_PRIVATE_
 
diff -Naur vstuff-1.1.2/libq931/Makefile.am vstuff-1.1.2.oden/libq931/Makefile.am
--- vstuff-1.1.2/libq931/Makefile.am	2008-11-05 15:49:11.000000000 +0100
+++ vstuff-1.1.2.oden/libq931/Makefile.am	2008-11-26 16:06:22.000000000 +0100
@@ -89,7 +89,7 @@
 	libq931/timer.h				\
 	libq931/util.h
 
-libq931_la_LDFLAGS = -module -version-info 1:0:0 -no-undefined
+libq931_la_LDFLAGS = -version-info 1:0:0 -no-undefined
 
 libq931_ladir = libq931
 
diff -Naur vstuff-1.1.2/libskb/Makefile.am vstuff-1.1.2.oden/libskb/Makefile.am
--- vstuff-1.1.2/libskb/Makefile.am	2008-11-05 15:49:11.000000000 +0100
+++ vstuff-1.1.2.oden/libskb/Makefile.am	2008-11-26 16:06:03.000000000 +0100
@@ -14,7 +14,7 @@
 libskb_la_SOURCES = libskb.c
 
 libskb_ladir = libskb
-libskb_la_LDFLAGS = -module -version-info 1:0:0 -no-undefined
+libskb_la_LDFLAGS = -version-info 1:0:0 -no-undefined
 
 nobase_include_HEADERS = libskb.h
 
